test/browser_tests/spec_suites/wagtail/wagtail-page-templates.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
```

Comment:
moved wagtail tests into its own folder to separate them from legacy static site tests for now